packages/@okta/vuepress-site/docs/guides/configure-user-scoped-account-management/main/index.md

```diff
@@ -0,0 +1,258 @@
+
+## About the Postman collection
+
+It is up to you how you set up users to call the MyAccount API to manage their account. In this guide, we provide sample API calls using a Postman collection to demonstrate them in a language/platform neutral way.
+
+> **Note:** To run the Postman collection, you need an end-user access token. Use a password grant type or [SDK](/docs/guides/auth-js/-/main/) to get the token.
+
+## API Versioning
+
+A valid API version in the `Accept` header is required to access the API. Current version V1.0.0
+
+```json
+Accept: application/json; okta-version=1.0.0
+```
+
+## Grant the required scopes
+
+> **Note:** If you have a custom Authorization Server, the MyAccount API-related scopes are granted by default. See [Create an Authorization Server](/docs/guides/customize-authz-server/-/main/).
+
+When you are ready to grant the required scopes, follow these steps:
+
+1. Sign in to your Okta organization with your administrator account and go to **Applications** > **Applications**.
+1. Open your OpenID Connect client app.
+1. In the **Okta API Scopes** tab, grant the following scopes:
+   * For access to both GET and POST/DELETE endpoints:
+      * `okta.myAccount.email.manage`
+      * `okta.myAccount.phone.manage`
+      * `okta.myAccount.profile.manage`
+   * For access to GET endpoints only:
+      * `okta.myAccount.email.read`
+      * `okta.myAccount.phone.read`
+      * `okta.myAccount.profile.read`
+
+
+## Add a secondary email
+
+<ApiOperation method="post" url="/idp/myaccount/emails" />
+
+In this use case, the end user adds a `SECONDARY` email address to their account. The new email address is in UNVERIFIED status.
+
+### Required scope and role
+
+An Okta scope of `okta.myAccount.email.manage` is required to use this endpoint.
+
+> **Note:** Admin users aren't allowed to call the `/idp/myaccount/emails` POST endpoint.
+
+### Request path parameters
+
+N/A
+
+### Request query parameters
+
+N/A
+
+### Request body
+
+This API requires a [My Email Request object](#my-email-request-object) as its request body.
+
+```bash
+{
+  "profile": {
+      "email" : "some.primary.email@okta.com"
+  },
+   "role": "SECONDARY",
+   "sendEmail": true
+}
+```
+
+#### Response body
+
+The requested [My Email object](#my-email-object)
+
+#### Error Responses
+
+If an invalid email is passed to `profile` in the request body, the response returns a 400 BAD REQUEST with error code E0000001.
+
+If the email operation isn't enabled for the request `role` on the org, the response returns a 403 FORBIDDEN with error code E0000038.
+
+If the email already exists for the current user, the response returns a 409 CONFLICT with error code E0000157.
+
+##### Request
```

Comment:
Is this a new section or it belongs to `## Add a secondary email` ? do we need a new section header?

Reply:
Same section as "secondary email". I need to adjust the headings.